src/DismissibleContext.tsx

```diff
@@ -151,23 +188,69 @@ export const useDismissibleContext = () => {
 }
 
 export const DISMISSIBLE_LOGGED_OUT_USER_ID = "user" as const
+export const PREFIX = `progressive-onboarding.dismissed`
 
 export const localStorageKey = (id: string) => {
-  return `progressive-onboarding.dismissed.${id}`
+  return `${PREFIX}.${id}`
+}
+
+const loadKeysFromStorage = (initialKeys: DismissibleKeys): DismissibleKeys => {
+  const allStorageKeys = Array.from(
+    { length: localStorage.length },
+    (_, index) => localStorage.key(index)
+  ).filter(Boolean) as string[]
+
+  const dynamicKeys = allStorageKeys.reduce<string[]>((acc, storageKey) => {
+    if (!storageKey.startsWith(PREFIX)) {
+      return acc
+    }
+
+    const key = localStorage.getItem(storageKey)
```

Comment:
Not SSR-safe

Reply:
ugh right, lemme fix real quick

Reply:
have grown lazy with the luxury of not having to worry about SSR in volt

Reply:
Fixed in here https://github.com/artsy/dismissible/pull/9